components/clp-package-utils/clp_package_utils/scripts/compress.py

```
@@ -162,11 +175,19 @@ def main(argv):
         logger.exception("Failed to load config.")
         return -1
 
+    storage_engine: StorageEngine = clp_config.package.storage_engine
+    dataset = parsed_args.dataset
+    if StorageEngine.CLP_S == storage_engine:
+        dataset = CLP_DEFAULT_DATASET_NAME if dataset is None else dataset
+    elif dataset is not None:
+        logger.error(f"Dataset selection is not supported for storage engine: {storage_engine}.")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)